_posts/en/2021-06-15-review-e-go-white-tiles.markdown

```diff
@@ -0,0 +1,113 @@
+
+![](/assets/images/review-e-go-white/001-012.jpg)
+
+The number tiles look pretty well.
+
+![](/assets/images/review-e-go-white/001-013.jpg)
+
+Trivia: in this set, the two, four, and six of bamboos are not centrally symmetric - one can see in which direction the bamboo sticks are pointed (just like in AMOS MAX). The majority of players won't notice this fact, but some may feel compelled to rotate them in order to align them with the rest.
+
+![](/assets/images/review-e-go-white/001-014.jpg)
+
+The circles look OK with one exception (can you see it yet?) - but more about it in the next chapter.
+
+## Traits, pros and cons
+
+The tiles are of uniform color, which means that discarding white dragons in a cultural way requires an additional tile flip; another quirky situation happens when a white dragon happens to be a dora indicator, which is impossible to tell apart from the situation where the indicator has not been revealed.
+
+These tiles do not have a weighted core and therefore are rather light when compared e.g. to the AMOS MAX tiles, which is heavily played in Kraków. All in all it's a net benefit, since the wooden box that contains the tiles weighs a lot; putting even more weight in it could cause some logistic issues. Lightweighted tiles also give much less noise during tile mixing, which can cause people who like the "sparrow chipping" sound to frown and the people who prefer to play without annoying their neighbors to sigh with relief.
+
+A huge upside of the set is that the tiles are *absolutely even* and building a wall from them is pure pleasure. It does not have a problem from the AMOS MAX tile set, where the tiles in the middle of the wall, despite being just as even, also weigh down a lot and can cause an airborne row of tiles to collapse; here, the wall can be raised and built even without a lot of experience in the art of tiling.
+
+![](/assets/images/review-e-go-white/002-001.jpg)
+
+A small downside of the set is that some tiles are dirty with paint that has not been cleanly removed during tile production; I have counted about a dozen of them. I have managed to clean some of them using cotton swabs with acetone (warning, don't rub too hard - it's possible to remove the paint from inside tile carvings!), but most of them were immune to my cleaning attempts.
+
+![](/assets/images/review-e-go-white/002-002.jpg)
+
+Other than for the dirtiness on the third-from-top four of circles, the second one has its carving off-center; other than that, one can see small cracks of the material nearby the carvings (second-from-top three of circles, two topmost fives of circles). These two issues are the ones I consider to be the greatest; paradoxically it's a plus, since all of the issues I found are practically unnoticeable during standard play.
```

Comment:
```suggestion
Other than for the dirtiness on the third-from-top four of circles, the second one has its carving off-center; furthermore, one can see small cracks of the material nearby the carvings (second-from-top three of circles, two topmost fives of circles). These two issues are the ones I consider to be the greatest; paradoxically it's a plus, since all of the issues I found are practically unnoticeable during standard play.
```

Sugeruję usunąć powtórzenie "other than".